community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/helpers/LRUCache.scala

```diff
@@ -21,10 +21,10 @@ package org.neo4j.cypher.internal.compiler.v2_2.helpers
 
 import com.googlecode.concurrentlinkedhashmap.ConcurrentLinkedHashMap
 
-class LRUCache[K, V](cacheSize: Int) extends ((K, => V) => V) {
+class LRUCache[K, V](val size: Int) extends ((K, => V) => V) {
```

Comment:
Nope. It's used [here](https://github.com/neo4j/neo4j/pull/6569/files#diff-d13b42573e66ee6724d02b58fbf36682R31)